ComposeSample/app/build.gradle

```diff
@@ -7,19 +7,14 @@
 
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-android-extensions'
 
 android {
     compileSdkVersion rootProject.ext.compileSdkVersion
-    buildToolsVersion rootProject.ext.buildToolsVersion
```

Comment:
You don't want to define the build tools version you will use? 
Following: https://developer.android.com/studio/releases/build-tools it will use the build tools that the gradle plugin use under the hood, but maybe we should maintain the control over which version are we using. Since it's defined in the `gradle.properties` file we can still use it.
If not needed anymore and we want to rely on the plugin, then I think we don't need that line in the `gradle.properties` file anymore.

Reply:
No, we don't need that. `If you're using Android plugin for Gradle 3.0.0 or higher, your project automatically uses a default version of the build tools that the plugin specifies.`